src/transformers/modeling_tf_pytorch_utils.py

```diff
@@ -66,10 +67,12 @@ def convert_tf_weight_name_to_pt_weight_name(tf_name, start_prefix_to_remove="",
     if len(tf_name) > 1:
         tf_name = tf_name[1:]  # Remove level zero
 
+    tf_weight_shape = list(tf_weight_shape)
+
     # When should we transpose the weights
-    if tf_name[-1] == "kernel" and tf_weight_shape is not None and tf_weight_shape.rank == 4:
+    if tf_name[-1] == "kernel" and tf_weight_shape is not None and len(tf_weight_shape) == 4:
```

Comment:
Why this change? (not saying it's unwarranted, just curious)

Reply:
Cause the object passed can be a regular shape, not necessarily a TensorFlow shape, and as such, does not always have a rank attribute.